docker-compose.example.yml

```
@@ -2,12 +2,11 @@ version: '3'
 
 services:
   woodpecker-server:
-    build:
-      context: .
-      dockerfile: Dockerfile
     image: woodpeckerci/woodpecker-server:local
```

That's `local`? Shouldn't it be `latest`?

```suggestion
    image: woodpeckerci/woodpecker-server:latest
```